src/app/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import Typography from '../../components/Typography/Typography';
+import { Link } from 'react-router-dom';
+import styles from './LoginPage.module.css';
+
+export default function LoginPage(): JSX.Element {
+  return (
+    <main className={styles.login}>
+      <h2>
+        <Typography size="l">Einloggen</Typography>
+      </h2>
+      <TextInput id="name" inputType="text">
+        Name
+      </TextInput>
+      <TextInput id="password" inputType="password">
+        Passwort
+      </TextInput>
+      <Link to="" className={styles.button}>
+        <SubmitButton icon="signup">Einloggen</SubmitButton>
+      </Link>
+      <Link to="/signupselection" className={styles.signupLink}>
+        <p>
+          <Typography size="s">
+            Noch keinen Account? <br />
+            Hier anmelden!
+          </Typography>
```

Comment:
I think this already returns a `<p>` nesting paragraphs is not allowed, because they [aren't allowed to contain block level elements](https://www.w3.org/TR/html401/struct/text.html#h-9.3.1)